src/Commands/core/BrowseCommands.php

```diff
@@ -28,11 +31,20 @@ class BrowseCommands extends DrushCommands
      */
     public function browse($path = '', $options = ['browser' => null])
     {
+        // TODO: Remove 2nd branch when no longer needed.
+        if ($this->hasSiteAliasManager()) {
+            $aliasRecord = $this->siteAliasManager()->getSelf();
+            $is_remote = $aliasRecord->isRemote();
+            $site_record = $aliasRecord->exportConfig()->get('options', []);
```

Comment:
Maybe we should have an API for this (`exportConfig()` + `get('options')`). Until we rewrite backend invoke, this pattern will probably appear a lot. Optional.